src/providers/facebook-login-provider.ts

```diff
@@ -45,6 +45,8 @@ export class FacebookLoginProvider extends BaseLoginProvider {
     user.name = response.name;
     user.email = response.email;
     user.token = response.token;
+    user.firstName = response.first_name;
+    user.lastName = response.lastName;
```

Comment:
I think it should be last_name, at least for v3.1 api.

Reply:
Ok, I'll fix it